data/json/vehicle_parts.json

```diff
@@ -1130,7 +1130,7 @@
       "repair": { "skills": [ [ "mechanics", 1 ] ], "time": 2000, "using": [ [ "adhesive", 1 ] ] },
       "removal": { "skills": [ [ "mechanics", 1 ] ], "qualities": [ { "id": "SCREW", "level": 1 } ] }
     },
-    "flags": [ "OPENABLE", "OPENCLOSE_INSIDE", "OPAQUE", "CURTAIN" ],
+    "flags": [ "OPENABLE", "OPENCLOSE_INSIDE", "OPAQUE", "CURTAIN", "WINDOW_CURTAIN" ],
```

Comment:
The info field from the WINDOW_CURTAIN flag will be displayed as part of the description, so you need to trim the description to remove the parts that are repeated here.

Reply:
Ok yeah I see how that works with the description now. I'll fix up the flags and descriptions to get them to make sense all together.